Classes/Converter/FormValueTimestampConverter.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace AOE\SchedulerTimeline\Converter;
+
+/*
+ * (c) 2020 AOE GmbH <dev@aoe.com>
+ *
+ * This file is part of the TYPO3 Scheduler Timeline Extension.
+ *
+ * It is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License, either version 2
+ * of the License, or any later version.
+ *
+ * For the full copyright and license information, please read the
+ * LICENSE.txt file that was distributed with this source code.
+ *
+ * The TYPO3 project - inspiring people to share!
+ */
+
+class FormValueTimestampConverter
+{
+    /**
+     * @param string $value
+     * @return string
+     */
+    public static function convertValueToStrToTimeValue(string $value)
+    {
+        $timeArray = preg_split('#(?<=\d)(?=[a-z])#i', $value);
+        return $timeArray[0] . ' ' . self::getFullLabel($timeArray[1]);
```

Comment:
not necessary, but small check whether set?
`$timeArray[0] ?? 'default value'`
`$timeArray[1] ?? 'default value'`